recipe/meta.yaml

```diff
@@ -45,6 +45,28 @@ outputs:
       strong:
         - libgfortran-ng >=4.9                                       # [linux]
         - libgfortran-ng >=3.0                                       # [osx]
+{% else %}
+  - name: toolchain_m2w64_c_{{ target_platform }}
+    requirements:
+      - m2w64-toolchain
+    run_exports:
+      strong:
+        - m2w64-gcc-libs-core
+
+  - name: toolchain_m2w64_cxx_{{ target_platform }}
+    requirements:
+      - m2w64-toolchain
+    run_exports:
+      strong:
+        - m2w64-gcc-libs-core
```

Comment:
Looks like `m2w64-gcc-libs` requires this and has `libstdc++`. So maybe we should be adding it here.